tests/token/erc20/test_ERC20_Burnable.py

```diff
@@ -123,9 +130,63 @@ async def test_burn_from_zero_address(erc20_factory):
 
 @pytest.mark.asyncio
 async def test_burn_invalid_uint256(erc20_factory):
-    erc20, _ = erc20_factory
+    erc20, _, _ = erc20_factory
 
     await assert_revert(
         erc20.burn(INVALID_UINT256).invoke(),
         reverted_with="ERC20: amount is not a valid Uint256"
     )
+
+
+@pytest.mark.asyncio
+async def test_burn_from(erc20_factory):
+    erc20, account1, account2 = erc20_factory
+
+    await signer.send_transaction(
+        account1, erc20.contract_address, 'increaseAllowance', [
+            account2.contract_address,
+            *AMOUNT
+        ])
+
+    await signer.send_transaction(
+        account2, erc20.contract_address, 'burnFrom', [
+            account1.contract_address,
+            *AMOUNT
+        ])
+
+    new_balance = sub_uint(INIT_SUPPLY, AMOUNT)
+
+    execution_info = await erc20.balanceOf(account1.contract_address).invoke()
+    assert execution_info.result.balance == new_balance
```

Comment:
good call, ill add these tests in

Reply:
@andrew-fleming  added testing for the event. looks like `burnFrom` reverts when burn amount exceeds balance is covered by the following test and having account as the zero address results in an underflow error since the zero address cant give any approval

Reply:
Ah my mistake. Thank you for adding in the event test!